source/ensdf/records/Continuation.cpp

```diff
@@ -222,27 +222,27 @@ parse_continuation(const std::string& crecs)
 }
 
 void merge_continuations(std::map<std::string, Continuation> &to,
-                         const std::map<std::string, Continuation>& from,
-                         const std::string &debug_line)
+                         const std::map<std::string, Continuation>& from)
+                         //const std::string &debug_line)
 {
   for (const auto& cont : from)
   {
     if (cont.first == "XREF")
       continue;
     if (!to.count(cont.first))
       to[cont.first] = cont.second;
-    else
-    {
-      if (to[cont.first].is_uncert()
-          && cont.second.is_uncert())
-      {
+    // else
```

Comment:
I would err in the other direction. Commented code tends to rot.
Maybe switch the print statement below from DBG to TRC to spam less?

Reply:
The call has been changed to TRC, but the logger doesn't know how to format the Continuation struct so I have left it commented while I look into that.

Reply:
Yeah, you probably need to either write an `fmt` interface for it or just some sort of `to_string()` function.

Reply:
Custom formatter added with a best guess for what should be printed.